website/news/embed.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
+include_once __DIR__ . '/../../config/config.inc.php';
 // in theory this file is no longer used
-
 include __DIR__ . '/../../config/news.inc.php';
```

Comment:
Awkward place to put the comment, I think. I'd do it like this:

```php
// in theory this file is no longer used

include_once __DIR__ . '/../../config/config.inc.php';
include __DIR__ . '/../../config/news.inc.php';
```

Reply:
Works for me.